.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: Linux Release
+
+on:
+  release: 
+    types:
+      - released
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build ${{ matrix.release }} ${{ matrix.arch }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        release:
+          - bullseye
+          - buster
+          - stretch
+          - groovy
+          - focal
+          - bionic
+
+        arch:
+          - amd64
+          - arm64
+          - armhf
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install make and mmv
+        run: sudo apt-get install make mmv
+
+      - name: Install Docker
+        uses: docker-practice/actions-setup-docker@v1
+
+      - name: Build
+        run: ./build ${{ matrix.release }} ${{ matrix.arch }} dist
+
+      - name: Upload Packages
+        uses: actions/upload-artifact@v2.2.3
+        with:
+          name: ${{ matrix.release }} ${{ matrix.arch }} package
+          path: dist
+
+  publish:
+    name: Push artifacts to repo
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          path: artifacts
+
+      - name: Display structure of downloaded files
+        run: ls -R
+
+      - name: Push files to repo
+        uses: appleboy/scp-action@v0.1.1
+        with:
+          host: ${{ secrets.REPO_HOST }}
+          username: ${{ secrets.REPO_USER }}
+          password: ${{ secrets.REPO_PASSWORD }}
+          port: 22
+          source: "artifacts/*"
+          target: "/srv/repository/incoming/ffmpeg/${{ github.event.release.tag_name }}"
+          strip_components: 2
```

Comment:
just curious, and not blocking but why scp instead of rsync (then again I only used rsync in the `android` PR due to a comment by ferferga on matrix)

Reply:
First time building a GH actions pipeline from scratch so combining them might be a good idea. I didn't think about that.
I used scp because i was googeling for a task like [this one](https://github.com/jellyfin/jellyfin/blob/master/.ci/azure-pipelines-package.yml#L66) and this task was the one that came up with the most stars. Is there an advantage with using rsync?

Reply:
I've always used `scp` personally. A bit simpler than rsync and we can't always guarantee `rsync` is there (if it is great).

Reply:
there probably is, I haven't played to much with it sadly yet, nor have I looked to deep into it. The sole reason I went for it was ferferga using it in a link he provided on matrix of his server CI revamp. (I'll look for the ref, but IMHO if it works :man_shrugging:)

Reply:
> I've always used `scp` personally. A bit simpler than rsync and we can't always guarantee `rsync` is there (if it is great).

In GH Actions on the [Ubuntu image](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) it is always there afaik

Reply:
@joshuaboniface I can change the workflow to use rsync like @ferferga [here](https://github.com/ferferga/jellyfin/blob/e06cdde3600cbf17a72fc70252067271fb74f82d/.github/workflows/build-packages.yml#L58). However, this action only works with a ssh private key instead of a password.
I have no preference for scp or rsync so I'll let you decide